viewing_party/party.py

```diff
@@ -1,23 +1,179 @@
+from collections import Counter
+from shutil import move
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {
+        "title":title,
+        "genre":genre,
+        "rating":rating
+    }
+
+    for val in movie_dict.values():
+        if not val:
+            return None

```

Comment:
A way to optimize this function would be to do this check first thing. If title, genre, rating are not truthy then we don't need to build a dictionary. 

You could add this before line 7:
```
if not title or not genre or not rating:
	     return None
```
This guard clause would then also get rid of the need to loop through the dictionary too. 

You can read more about guard clauses here if you're interested: https://deviq.com/design-patterns/guard-clause